backend/src/main/java/com/kk/heapoverflow/controller/QuestionController.java

```diff
@@ -19,13 +23,18 @@ public class QuestionController {
 
     private final QuestionService questionService;
 
-    @GetMapping("/getById")
-    public Question getById(@RequestParam Long id) {
+    @GetMapping("/{id}")
+    public QuestionByIdDto getById(@PathVariable Long id) {
         return questionService.getQuestionById(id);
     }
 
     @GetMapping("/getPreviewPage")
     public QuestionPreviewPageResponseDto getQuestionsPreviewPage(Pageable pageable) {
         return questionService.getQuestionsPreviewPage(pageable);
     }
+
+    @PostMapping("/new")
```

Comment:
Only using post method is clear that we will create something on the server, no need to add “new” on path
```suggestion
    @PostMapping
```